CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 # cmake based build of Subsurface
 
+
```

Comment:
why are we adding another new line here? That seems like leftover from some other changes maybe?

Reply:
Corrected.